packages/utils/src/arbeidsgiverUtils.ts

```diff
@@ -0,0 +1,28 @@
+import { dateFormat } from './dateFormat';
```

Comment:
Det var ein gong sånn at utils-pakka skulle vera mest mogleg generell, og ikkje ha domene-logikk. Men eg veit ikkje om det gjer meining lenger... Verkar unødvendig å måtte laga eige pakke for dette. Eg berre nemner det